src/pages/instances/InstanceOverview.tsx

```diff
@@ -31,162 +38,282 @@ const InstanceOverview: FC<Props> = ({ controlTarget, instance }) => {
   });
 
   if (metricError) {
-    notify.failure("Could not load metrics.", metricError);
+    setInTabNotification(failure("Could not load metrics.", metricError));
+  }
+
+  const instanceMetrics = getInstanceMetrics(metrics, instance);
+
+  const {
+    data: networks = [],
+    error: networkError,
+    isLoading: isNetworkLoading,
+  } = useQuery({
+    queryKey: [queryKeys.networks, instance.project],
+    queryFn: () => fetchNetworks(instance.project),
+  });
+
+  if (networkError) {
+    setInTabNotification(failure("Could not load networks.", networkError));
+  }
+
+  const instanceNetworks = Object.values(instance.expanded_devices)
+    .filter(isNicDevice)
+    .map((network) => network.network);
+
+  const networksHeaders = [
+    { content: "Name", sortKey: "name" },
+    { content: "Type", sortKey: "type" },
+    { content: "Managed", sortKey: "managed" },
+    { content: "IPV4", className: "u-align--right" },
+    { content: "IPV6" },
+  ];
+
+  const networksRows = networks
+    .filter((network) => instanceNetworks.includes(network.name))
+    .map((network) => {
+      return {
+        columns: [
+          {
+            content: (
+              // TODO: fix this link to point to the network detail page
+              <Link
+                to={`/ui/${instance.project}/networks`}
+                title={network.name}
+              >
+                {network.name}
+              </Link>
+            ),
+            role: "rowheader",
+            "aria-label": "Name",
+          },
+          {
+            content: network.type,
+            role: "rowheader",
+            "aria-label": "Type",
+          },
+          {
+            content: network.managed ? "Yes" : "No",
+            role: "rowheader",
+            "aria-label": "Managed",
+          },
+          {
+            content: network.config["ipv4.address"],
+            className: "u-align--right",
+            role: "rowheader",
+            title: network.config["ipv4.address"],
+            "aria-label": "IPV4",
+          },
+          {
+            content: network.config["ipv6.address"],
+            role: "rowheader",
+            title: network.config["ipv6.address"],
+            "aria-label": "IPV6",
+          },
+        ],
+        sortData: {
+          name: network.name,
+          type: network.type,
+          managed: network.managed,
+        },
+      };
+    });
+
+  const {
+    data: profiles = [],
+    error: profileError,
+    isLoading: isProfileLoading,
+  } = useQuery({
+    queryKey: [queryKeys.profiles, instance.project],
+    queryFn: () => fetchProfiles(instance.project),
+  });
+
+  if (profileError) {
+    setInTabNotification(failure("Could not load profiles.", profileError));
   }
 
+  const profileHeaders = [
+    { content: "Name", sortKey: "name" },
+    { content: "Description", sortKey: "description" },
+  ];
+
+  const profileRows = profiles
```

Comment:
The default display should be in order from the `instance.profiles, not in order of the profile list response.

Maybe extract the profile table with the extra query into a dedicated component, same for the networks table.

Reply:
I forgot to mention: the network table doesn't list the up addresses, at least on the demo server the cells were always empty.

Reply:
> I forgot to mention: the network table doesn't list the IP addresses, at least on the demo server the cells were always empty.

It does, it's just that almost all the instances on the demo server have been assigned a disconnected network device. Instances with `lxdbr0` have the IP addresses populated: [one example](https://lxd-ui-248.demos.haus/ui/default/instances/detail/testing123).

Reply:
The IP's are sometimes missing. I.e. This [overview](https://lxd-ui-248.demos.haus/ui/default/instances/detail/alex-is-testing) has no IP on the fan network listed. If we check the `instance.state.networks.eth0` (which responds to the `lxdfan0` network name of `eth0`, then we see there is an ipv4 and also an ipv6 listed.
So the logic for picking the addresses to show in the overview tab should rely on the `instance.state.networks`. Though not sure if this is exhaustive, or if further addresses could be under the `instance.expanded_devices`, might be worth to ask it to the core team.

Reply:
I don't understand. The IPs that are being shown here are supposed to match the ones shown in the [Networks](https://lxd-ui-248.demos.haus/ui/default/networks) page, right? This at least is what I can reckon by looking at the HFM. So it seems just right as it is to me.

Reply:
The network page is also broken in that regard. We should focus to get the instance overview tab fixed in this PR.